app/test/publisher/api_test.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:gcloud/db.dart';
+import 'package:shelf/shelf.dart' as shelf;
+import 'package:test/test.dart';
+
+import 'package:pub_dartlang_org/publisher/models.dart';
+
+import '../shared/handlers_test_utils.dart';
+import '../shared/test_models.dart';
+import '../shared/test_services.dart';
+
+void main() {
+  group('Publisher API', () {
+    group('Update description', () {
+      // invoke the API endpoint
+      Future<shelf.Response> update(String publisherId,
+          {Map<String, dynamic> jsonBody, String authToken}) async {
+        return await httpRequest('PUT', '/api/publishers/$publisherId',
```

Comment:
No rush, but ideally, we should just use the generated client we made... this is what it's for...

Reply:
Good point. I guess I'll do it in a subsequent PR.